index.html

```diff
@@ -1959,22 +1959,27 @@ <h2>Forms</h2>
 
                         <br><br>
 
-                        <div class="form-check">
-                            <label class="form-check-label">
+                        <div class="form-check form-check-radio">
+                            <label class="form-check-label" for="exampleRadios1">
```

Comment:
AFAIK `for` is not needed when the related `input` is a label child.

Reply:
You're right :scream: